index.ts

```diff
@@ -18,6 +18,28 @@ const dbConfig = {
     },
 };
 
+const TIME_UNITS = {
+    's': 'SECOND',
+    'M': 'MINUTE',
+    'H': 'HOUR',
+    'd': 'DAY',
+};
+
+const getCronAndSqlInterval = (time: string): [string, string] => {
+    const unit = time.slice(-1);
+    const value = parseInt(time);
+    const cronUnit = TIME_UNITS[unit];
+    
+    if (!cronUnit) {
+        throw new Error('Invalid time format');
+    }
+
+    const cronTime = unit === 's' ? `*/${value} * * * *` : unit === 'd' ? `0 0 */${value} * *` : `0 */${value} * * *`;
+    const sqlInterval = `INTERVAL ${value} ${cronUnit}`;
+    
+    return [cronTime, sqlInterval];
+};
+
```

Comment:
This extraction and implementation can be kept in the PR, all the other stuff is unnecessarily extracted for a such a simple bot.

Reply:
I have the `getCronAndSqlInterval` function implemented in the code.

Reply:
Yea just keep that code, i dont see the point of all the other extractions